components/net/fetch/methods.rs

```diff
@@ -89,6 +89,11 @@ pub type DoneChannel = Option<(Sender<Data>, Receiver<Data>)>;
 /// [Fetch](https://fetch.spec.whatwg.org#concept-fetch)
 pub fn fetch(request: &mut Request, target: Target, context: &FetchContext) {
     fetch_with_cors_cache(request, &mut CorsCache::new(), target, context);
+    context
+        .timing
+        .lock()
+        .unwrap()
+        .set_attribute(ResourceAttribute::FetchStart);
```

Comment:
This needs to happen before calling `fetch_with_cors_cache` or it will execute after the fetch is complete.